server/schemas/buoy.py

```diff
@@ -0,0 +1,48 @@
+from typing import Any, Optional
```

Comment:
Don't understand what this file is meant for. As well as don't know what schemas is for? Definition of table architectures?

Reply:
Exactly! It's referred to as a Data Transfer Object - or DTO for short. You read more about them [here](https://www.okta.com/identity-101/dto/).

This is where [Pydantic](https://pydantic-docs.helpmanual.io/) is used in our stack to validate our data; it uses this schema as the blueprint.

Reply:
So schemas is a directory filled with unique DTOs to simplify CRUD operations of data between client and DB? Also still not 100% sure what pydantic does. On my side pydantic keeps throwing fits on VS Code.